sdk/synapse/azure-analytics-synapse-accesscontrol/src/main/java/com/azure/analytics/synapse/accesscontrol/AccessControlClient.java

```diff
@@ -0,0 +1,134 @@
+
+import com.azure.analytics.synapse.accesscontrol.implementation.AccessControlClientImpl;
+import com.azure.analytics.synapse.accesscontrol.models.ErrorContractException;
+import com.azure.analytics.synapse.accesscontrol.models.RoleAssignmentDetails;
+import com.azure.analytics.synapse.accesscontrol.models.RoleAssignmentOptions;
+import com.azure.analytics.synapse.accesscontrol.models.SynapseRole;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.http.rest.PagedIterable;
+import java.util.List;
+
+/** Initializes a new instance of the synchronous AccessControlClient type. */
+@ServiceClient(builder = AccessControlClientBuilder.class)
+public final class AccessControlClient {
+    private final AccessControlClientImpl serviceClient;
+
+    /** Initializes an instance of AccessControlClient client. */
+    AccessControlClient(AccessControlClientImpl serviceClient) {
+        this.serviceClient = serviceClient;
+    }
+
+    /**
+     * Create role assignment.
+     *
+     * @param createRoleAssignmentOptions Role Assignment request details.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws ErrorContractException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return role Assignment response details.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public RoleAssignmentDetails createRoleAssignment(RoleAssignmentOptions createRoleAssignmentOptions) {
+        return this.serviceClient.createRoleAssignment(createRoleAssignmentOptions);
+    }
+
+    /**
+     * List role assignments.
+     *
+     * @param roleId Synapse Built-In Role Id.
+     * @param principalId Object ID of the AAD principal or security-group.
+     * @param continuationToken Continuation token.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws ErrorContractException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return a list of role assignments.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public List<RoleAssignmentDetails> getRoleAssignments(String roleId, String principalId, String continuationToken) {
+        return this.serviceClient.getRoleAssignments(roleId, principalId, continuationToken);
+    }
+
+    /**
+     * List role assignments.
+     *
+     * @throws ErrorContractException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return a list of role assignments.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public List<RoleAssignmentDetails> getRoleAssignments() {
```

Comment:
Should the Api's returning a  `List` be updated to use prefix `list..` according to [this](https://azure.github.io/azure-sdk/java_design.html#common-service-client-patterns)?